### ISSUE TITLE
[Accessibility]: SVG element has no accessible name error for the for all charts

### ISSUE THREAD

**Issue by Chinmaya-812** [carbon-design-system/carbon-charts#1946]
### Application/Team

Cloud pak for data / Zen

### Environment

Automated testing tool and ruleset

### What happened? What did you expect to happen instead?

Getting ‘The SVG element has no accessible name’ accessibility issue on the charts. i tried adding Aria-label, title, tooltip to the gaugeChart & linechart tag, but it is not helping to fix the issue. i have tried the codesandbox from the carbondesignsystems webpage and see the error there as well while running the accessibility tool: https://stackblitz.com/edit/react-huv4phmk?file=src%2Foptions.js. 

https://stackblitz.com/edit/react-huv4phmk?file=src%2Foptions.js,package.json,src%2Findex.js

This SVG accessibility violation is comes almost all component of the @carbon/charts-react. 
I just add codesandbox link for 

![Image](https://github.com/user-attachments/assets/2c767c73-e4fc-49af-873a-a28e41b38a3f)

### What WCAG 2.1 checkpoint does the issue violate?

1.1.1 Non-text Content

### Version

    "@carbon/charts-react": "1.22.14"

### Data & options used

```js

```

### Relevant log output

```js

```

### StackBlitz repro

https://stackblitz.com/edit/react-huv4phmk?file=src%2Foptions.js

https://stackblitz.com/edit/react-huv4phmk?file=src%2Foptions.js,package.json,src%2Findex.js

### What priority level would this issue be in your opinion?

P1
